### PR TITLE
Master window websocket handling test

### PR DESCRIPTION
**Pull request by siemiatj**
Related to #2645

### CODE REVIEW

**Comment by siemiatj** [comment]
Waiting for bump of node version in Jenkins to get a green build

File: src/utils/websockets.js [hunk 1]
```diff
@@ -64,7 +64,13 @@ export function connectWS(topic, onMessageCallback) {
     };
     /*eslint-enable no-console */
 
-    this.sockClient.activate({}, subscribe);
```
Comment (petrican) [inline src/utils/websockets.js]: Why do you remove the subscribe callback? Do you want to leave it only onConnect hook ?

**Comment by TheBestPessimist** [comment]
Jenkins still fails here, even with node 13: https://jenkins.metasfresh.com/job/metasfresh-webui-frontend/job/master-window-tests/6/console

**Comment by petrican** [comment]
> jenkins build fails even with node 13 https://jenkins.metasfresh.com/job/metasfresh-webui-frontend/job/master-window-tests/6/console

@siemiatj  I see you already updated  jsdom and set jest-environment-jsdom-sixteen. Not sure what is going on this needs to be investigated a bit. Thanks

File: src/utils/websockets.js (hunk 1 above)
Comment (siemiatj) [reply src/utils/websockets.js]: Because it doesn't take a callback: https://stomp-js.github.io/api-docs/latest/classes/Client.html#activate

**Pull request merged**